src/app/challenges/[...id]/page.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { auth } from "@/auth";
+import { notFound, redirect } from "next/navigation";
+import { DashboardLayout } from "@/app/components/layout/dashboard";
+import NestedChallanges from "@/app/components/sections/nested-challenges";
```

Comment:
```suggestion
import NestedChallenges from "@/app/components/sections/nested-challenges";
```

There is a typo here in component name.